p5/sketch/Skia2DRenderer/handlers.py

```diff
@@ -0,0 +1,67 @@
+import builtins
+from os import environ
+import glfw
+from p5.core import p5
+from p5.sketch.events import MouseEvent
+
+"""
+This file contains the handlers needed for glfw event handling
+"""
+
+# TODO : Make a raw_event class and populate it with data about current events
+# TODO : Design of raw_event should follow that of Vispy's raw event beacuse events.py is highly based on it  
+
+BUTTONMAP = {glfw.MOUSE_BUTTON_LEFT: 1,
+                glfw.MOUSE_BUTTON_RIGHT: 2,
+                glfw.MOUSE_BUTTON_MIDDLE: 3
+                }
+
+class PseudoMouseEvent:
```

Comment:
@ziyaointl I am using a `PseudoEventClass` to mimic the data that was generated and need while using vispy-based events because the events.py is highly dependent on the `event` object passed by vispy. 

Please share your thoughts on this.

Reply:
I think this is solution has the pro of modifying minimal code in the current codebase, but I think in terms of keeping abstractions between renderers clean we may want to not include information about vispy inside of the skia renderer.

One solution I propose is to first modify the `Event`, `MouseEvent`, and `KeyEvent` classes in p5py so they don't directly rely on the raw events emitted by vispy, but rather specific information relavant from those events (e.g. `event.pos, event.buttons`). Then, one could write two helper functions that translate either vispy events or glfw events to p5py events. This seems to be similar to what @mrdaybird wrote in his proposal.

Let me your thoughts! If you also think this is a good idea, you can either create a new PR that refactors the existing event classes and rebase this skia PR on top of it, or you can keep the `PseudoMouseEvent` for now and create the refactor PR later down the road :)

Reply:
Apologies for the late response, I was shifting back home for summer break. 

Yes, this is a good idea. We should refactor the event classes. I will add a helper function in `util` to generate a p5py event according to the `builtins.current_renderer` we are using.  

I will keep the `PseudoMouseEvent` now as it is working fine now and I could quickly move to the renderer implementation. 

Also, the Renderer and Events modules are completely independent of each other so I am thinking of refactoring the events module later.

Reply:
No worries at all! I think putting helper functions in either `events.py` or the respective renderers may be slightly better than putting them into the global util file, but that's just my opionon. The rest of the plan sounds great. Thank you!